lib/swaggerValidator/trafficValidator.ts

```diff
@@ -19,12 +29,24 @@ export interface RuntimeException {
   message: string;
 }
 
+export interface OperationCoverageInfo {
+  readonly spec: string;
+  readonly coveredOperaions: number;
```

Comment:
Rename coveredOperations to testedOperations. 

Add testPassOperations which count the operation if (validationResult.responseValidationResult.isSuccessful === true && validationResult.requestValidationResult.isSuccessful === true)